sdks/python/apache_beam/utils/counters.py

```diff
@@ -134,6 +134,8 @@ class Counter(object):
   # Distribution Accumulator Fn
   DISTRIBUTION = cy_combiners.DistributionCounterFn()
 
+  DISTRIBUTION_METADATA = cy_combiners.DistributionMetadataCounterFn()
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Please add comment; please also add comment indicating this is an experimental API with no backwards compatibility guarantees.

Reply:
https://github.com/apache/beam/blob/master/sdks/python/apache_beam/utils/counters.py#L23
We have declared 'no backwards compatibility guarantees'  at the beginning of counter.py